src/node_crypto.cc

```diff
@@ -2590,10 +2591,15 @@ void CipherBase::Init(const char* cipher_type,
                                1,
                                key,
                                iv);
+  CHECK_NE(key_len, 0);
```

Comment:
According to https://www.openssl.org/docs/man1.0.2/crypto/EVP_BytesToKey.html, this method returns 0 when an error is encountered. Don't you think we should instead use an `if` statement here and throw something if `key_len` is 0?

Reply:
Mhh yeah, in theory, yes. Not sure which error conditions could cause that, so maybe I will defer that to another PR. This change won't make the situation any worse :-)

Reply:
Agreed. Totally fine by me, just wanted to make sure.